<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Interface</class>
 <widget class="QWidget" name="Search_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="Search_Text">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>15</y>
     <width>730</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="placeholderText">
    <string>请输入搜索内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Resource/Back_Page.png</normaloff>:/Resource/Back_Page.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_Button">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Resource/Search.png</normaloff>:/Resource/Search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="Select_Platform">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
